cli/tools/standalone.rs

```diff
@@ -233,67 +228,6 @@ pub async fn write_standalone_binary(
   Ok(())
 }
 
-/// Transform the flags passed to `deno compile` to flags that would be used at
-/// runtime, as if `deno run` were used.
-/// - Flags that affect module resolution, loading, type checking, etc. aren't
-///   applicable at runtime so are set to their defaults like `false`.
-/// - Other flags are inherited.
-pub fn compile_to_runtime_flags(
```

Comment:
This was a huge distraction. We can just remove it.

Reply:
I'm not sure, what are we passing now?

Reply:
See https://github.com/denoland/deno/pull/16990/files#diff-388fb4e27905632c538ee418fcd8a62c38af40dc4321f402c795e6a08f9ca64fL130